src/modules/systemlib/system_params.c

```diff
@@ -37,6 +37,8 @@
  * System wide parameters
  */
 
+#ifdef __PX4_QURT
```

Comment:
Using a QuRT define here is pointless because its not being evaluated. It can be removed.